packages/utils/fluid-main/package.json

```diff
@@ -0,0 +1,96 @@
+{
+  "name": "@fluid-experimental/fluid-main",
```

Comment:
Might want to think of a better name?  Currently, it looks like it include runtime libraries only?  What is the plan for loader/driver things?

Reply:
Good question. I think we'll ultimately put more things in this library. But I could imagine us having a hosting library and a runtime library.